config/package_resouce.go

```diff
@@ -14,26 +14,229 @@
 
 package config
 
-import agentendpointpb "github.com/GoogleCloudPlatform/osconfig/internal/google.golang.org/genproto/googleapis/cloud/osconfig/agentendpoint/v1alpha1"
+import (
+	"errors"
+	"fmt"
+
+	agentendpointpb "github.com/GoogleCloudPlatform/osconfig/internal/google.golang.org/genproto/googleapis/cloud/osconfig/agentendpoint/v1alpha1"
+	"github.com/GoogleCloudPlatform/osconfig/packages"
+)
 
 type packageResouce struct {
 	*agentendpointpb.ApplyConfigTask_Config_Resource_PackageResource
+
+	policy ManagedPackage
```

Comment:
Should this be named managedPackage?

Reply:
That's the type, which is public (will be used by agentendpoint package)

Reply:
ack, originally I was using this field for something slightly different thus the odd naming, changed